client/blocks/reader-full-post/style.scss

```diff
@@ -189,8 +154,15 @@
 }
 
 .reader-full-post .back-button {
-	@include breakpoint-deprecated( "<660px" ) {
-		display: none;
+	display: none;
+	position: absolute;
+	top: 0;
+	left: 0;
+	right: unset;
```

Comment:
re: adjusting the back button positioning this is where that is currently set (relative absolute on the content frame)